pkgs/applications/graphics/gimp/default.nix

```diff
@@ -85,6 +94,7 @@ in stdenv.mkDerivation (finalAttrs: {
     gettext
     makeWrapper
     gtk-doc
+    libxslt
```

Comment:
Why is libxslt needed?

Reply:
I am not quite sure. When I applied my patch, it failed to build some menu XMLs without `xsltproc`

Reply:
If you built from git instead of tarball, that might be the problem. Probably better to include it for completeness but better do that in a separate commit.

Reply:
Not my PR so CC @MathiasSven ^^

Reply:
Not entirely sure what you mean, the source is the gimp tarball. I tried building it as well without `xsltproc` and it did not work. The tarball seems to come with pre-generated `xml` files under `./menus`, but when we add AVIF/HEIC, those (some?) must be regenerated. 

I will split the PR into two commits, but it does seem that `xsltproc` is required. I couldn't follow the Makefile completely, so I am not sure.